data/mm/macros.yml

```diff
@@ -14,7 +13,7 @@
 "can_use_elegy2": "can_play(SONG_EMPTINESS) && (has(MASK_ZORA) || has(MASK_GORON))"
 "can_use_elegy3": "can_play(SONG_EMPTINESS) && has(MASK_ZORA) && has(MASK_GORON)"
 "has_bombchu": "has(BOMB_BAG)"
-"has_beans": "event(MAGIC_BEANS)"
+"has_beans": "event(MAGIC_BEANS) || has(MAGIC_BEAN)"
```

Comment:
Not correct - you might have found the single bean but be unsable to buy more (because no deku mask)

Reply:
i originally set it that way, but figured just having the one bean would be enough for anything. I have just remembered the Well throws a wrench in that idea.

Reply:
Well + have to assume worst case usage as SoT resets beans to zero and you can also plant the wrong one